api/grpcserver/transaction_service.go

```diff
@@ -279,7 +277,7 @@ func (s TransactionService) TransactionsStateStream(in *pb.TransactionsStateStre
 						if in.IncludeTransactions {
 							tx, err := s.conState.GetMeshTransaction(txid)
 							if err != nil {
-								s.logger.Error("could not find transaction %v from layer %v: %v", txid, layer, err)
+								ctxzap.Error(stream.Context(), "could not find transaction from layer", txid.Field().Zap(), layer.Field().Zap(), zap.Error(err))
```

Comment:
Same as above, `txid` should in my oppinion implement `zapcore.ObjectMarshaler` instead of extending `log.Fied`